src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPropertySymbol.cs

```diff
@@ -50,7 +50,7 @@ public SynthesizedRecordPropertySymbol(
 
         public override IAttributeTargetSymbol AttributesOwner => BackingParameter as IAttributeTargetSymbol ?? this;
 
-        internal override bool IsCallerUnsafe => false;
+        internal override CallerUnsafeMode CallerUnsafeMode => CallerUnsafeMode.None;
```

Comment:
Do we still use `SynthesizedRecordPropertySymbol`s when the user provides an override of the defaults? For example:

```cs
record R(int I)
{
    public unsafe int I { get; set; } = I;
}
```